src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -183,10 +185,13 @@ public static Set<Tag> parseTags(Collection<String> tags) throws ParseException
     /**
      * Returns the String eventTitle passed to it.
      */
```

Comment:
I think the comment can be changed to:

```suggestion
    /**
     * Parses {@code eventTitle} into a {@code EventTitle}.
     */
```